### DISCUSSION TITLE
Vapor Deploy problem with google credentials

### DISCUSSION THREAD

**Discussion by amir-daneshkar-dev** [imdhemy/laravel-in-app-purchases#260] [Q&A] [answered]
Hi,
I'm using vapor for deploying to AWS and the env variable for **google_application_credentials** keeps throwing an error like  "could not find the file".
I fixed this problem on my local env (developing via sail) by adding the absolute path to env file (/var/www/html/google_application_credntials.json).
but I can't do that in vapor.

Update :
in config/liap.php, set` 'google_application_credentials' => base_path('google_application_credentials.json'),`

**Comment by imdhemy** [comment, marked as answer]
Hi 👋

What about trying one of the ideas mentioned in this comment?

https://github.com/imdhemy/laravel-in-app-purchases/issues/196#issuecomment-1310143260

**Reply by amir-daneshkar-dev** [reply]
Thanks, this worked.
Ill update the question for future ref.

**Reply by imdhemy** [reply]
My pleasure! 🙌
